apps/etterlatte-migrering/src/main/kotlin/migrering/MigrerSpesifikkSak.kt

```diff
@@ -39,12 +52,53 @@ internal class MigrerSpesifikkSak(
             return
         }
 
+        val pesyssak = hentSak(sakId).tilVaarModell().also {
+            pesysRepository.lagrePesyssak(pesyssak = it)
+        }
+        packet.eventName = Migreringshendelser.MIGRER_SAK
+        val request = pesyssak.tilMigreringsrequest()
+        packet.hendelseData = request
+        try {
+            verifiserer.verifiserRequest(request)
+        } catch (e: Exception) {
+            logger.warn("Sak ${request.pesysId} har ufullstendige data i PDL, kan ikke migrere")
+            pesysRepository.oppdaterStatus(request.pesysId, Migreringsstatus.FEILA)
```

Comment:
Ref. kommentaren over, så tror jeg det er nyttig å kunne hente ut data om hva som feilet og i hvilke steg, så vi slipper å rote for mye i loggene når vi tester på større datamengder.